app/controllers/users/delete_controller.rb

```diff
@@ -52,13 +53,21 @@ def send_push_notifications
       PushNotification::HttpPush.deliver(event)
     end
 
-    # rubocop:disable IdentityIdp/MailLaterLinter
     def notify_user_via_email_of_deletion
       current_user.confirmed_email_addresses.each do |email_address|
         UserMailer.with(user: current_user, email_address: email_address).
-          account_delete_submitted.deliver_now
+          account_delete_submitted.deliver_now_or_later
+      end
+    end
+
+    def notify_user_via_text_of_deletion
+      phone_numbers = MfaContext.new(current_user).phone_configurations.map(&:phone)
```

Comment:
I haven't dug too closely, but do we need to either:

- Filter to check for confirmed phone numbers (i.e. similar to "confirmed" email addresses above)
- Check for SMS capability on a given phone number